.CMake/compiler_opts.cmake

```diff
@@ -96,7 +96,6 @@ if(CMAKE_C_COMPILER_ID MATCHES "Clang")
     add_compile_options(-Wextra)
     add_compile_options(-Wpedantic)
     add_compile_options(-Wno-unused-command-line-argument)
-    add_compile_options(-Wno-language-extension-token)
```

Comment:
Removed this option from here because it's already included in build. See [this](https://github.com/open-quantum-safe/liboqs/pull/1470/commits/639977a0feb4bf139af94cd3d4553bad42296a07#r1264582142) comment.